api/v1alpha1/scheduledscaling_types.go

```diff
@@ -0,0 +1,132 @@
+}
+
+type ScheduledScalingPhase string
+
+//+kubebuilder:object:root=true
+//+kubebuilder:subresource:status
+
+// ScheduledScaling is the Schema for the scheduledscalings API
+type ScheduledScaling struct {
+	metav1.TypeMeta   `json:",inline"`
+	metav1.ObjectMeta `json:"metadata,omitempty"`
+
+	Spec   ScheduledScalingSpec   `json:"spec,omitempty"`
+	Status ScheduledScalingStatus `json:"status,omitempty"`
+}
+
+//+kubebuilder:object:root=true
+
+// ScheduledScalingList contains a list of ScheduledScaling
+type ScheduledScalingList struct {
+	metav1.TypeMeta `json:",inline"`
+	metav1.ListMeta `json:"metadata,omitempty"`
+	Items           []ScheduledScaling `json:"items"`
+}
+
+type TargetRefs struct {
+	// ScaleTargetRef is the target of scaling.
+	// It should be the same as the target of HPA.
+	ScaleTargetRef CrossVersionObjectReference `json:"scaleTargetRef" protobuf:"bytes,1,name=scaleTargetRef"`
+	//Tortoise to be targeted for scheduled scaling
+	TortoiseName *string `json:"tortoiseName,omitempty" protobuf:"bytes,2,name=tortoiseName"`
+}
+
+// CrossVersionObjectReference contains enough information toet identify the referred resource.
+type CrossVersionObjectReference struct {
+	// kind is the kind of the referent; More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#types-kinds
+	Kind *string `json:"kind,omitempty" protobuf:"bytes,1,opt,name=kind"`
+
+	// name is the name of the referent; More info: https://kubernetes.io/docs/concepts/overview/working-with-objects/names/#names
+	Name *string `json:"name,omitempty" protobuf:"bytes,2,opt,name=name"`
+
+	// apiVersion is the API version of the referent
+	// +optional
+	APIVersion *string `json:"apiVersion,omitempty" protobuf:"bytes,3,opt,name=apiVersion"`
+}
+
+type Schedule struct {
+	/// The schedule in Cron format, see https://en.wikipedia.org/wiki/Cron.
+	// start of schedule
+	StartAt *string `json:"startAt,omitempty" protobuf:"bytes,1,opt,name=startAt"`
+	// end of schedule
+	FinishAt *string `json:"finishAt,omitempty" protobuf:"bytes,2,name=finishAt"`
+}
+
+type Strategy struct {
+	// Resource scaling requirements
+	Static *Static `json:"static,omitempty" protobuf:"bytes,1,opt,name=static"`
+}
+
+type Static struct {
+	// Min replicas to be deployed on schedule
+	MinimumMinReplicas *int `json:"minimumMinReplica,omitempty" protobuf:"bytes,1,opt,name=minimumMinReplica"`
+	// Resources requested per container
+	MinAllocatedResources []ContainerResourceRequests `json:"minAllocatedResourcesomitempty,omitempty" protobuf:"bytes,2,opt,name=minAllocatedResources"`
+}
+
+// Not sure if I need this here, duplicated code from tortoise types
+type ContainerResourceRequests struct {
+	// ContainerName is the name of target container.
+	ContainerName string          `json:"containerName" protobuf:"bytes,1,name=containerName"`
+	Resource      v1.ResourceList `json:"resource" protobuf:"bytes,2,name=resource"`
+}
+
```

Comment:
Can you try to just import and use v1beta3.ContainerResourceRequests at L120. I guess it works

```suggestion
```